server/py/services/api/launcher.py

```diff
@@ -326,7 +326,7 @@ def enrich_runtime(
 
         # ensure the runtime has a project before we enrich it with the project's spec
         runtime.metadata.project = (
-            project_name or runtime.metadata.project or mlrun.mlconf.default_project
+            project_name or runtime.metadata.project or mlrun.mlconf.active_project
```

Comment:
also here, IIUC the active project is valid in the client, but the server shouldn't enrich to it (needs an explicit project)